core/src/test/java/org/apache/calcite/sql/parser/SqlParserTest.java

```diff
@@ -3699,6 +3700,13 @@ void checkPeriodPredicate(Checker checker) {
         .ok(expected);
   }
 
+  @Test public void testInsertWithRowDialect() {
+    final String expected = "INSERT INTO `emps`\n"
```

Comment:
Maybe we can rename this method to `testRowValueExpression` and we should test dialect that allows/not allow explicit `ROW` keywords.